migrations/234_submissions__regradable__add.sql

```diff
@@ -1,4 +1,4 @@
-ALTER TABLE submissions ADD COLUMN IF NOT EXISTS regradable boolean NOT NULL DEFAULT FALSE;
+ALTER TABLE submissions ADD COLUMN IF NOT EXISTS regradable boolean DEFAULT FALSE;
 
 UPDATE submissions
 SET regradable = (graded_at IS NOT NULL) OR (grading_requested_at IS NOT NULL);
```

Comment:
@mwest1066 If the concern is updating the whole table, isn't this a problem too?